DapperDoodle/ConfigurationHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using DapperDoodle.Interfaces;
+using Microsoft.Data.Sqlite;
+using Microsoft.Extensions.DependencyInjection;
+using MySql.Data.MySqlClient;
+
+namespace DapperDoodle
+{
+    public static class ConfigurationHelpers
+    {
+        public static void ConfigureDapperDoodle(
+            this IServiceCollection services, 
+            string connectionString,
+            DBMS dbms)
+        {
+            if (services is null)
+                throw new NullReferenceException("The service collection Specified is invalid");
+            
+            ConfigureCqrsInterfaces(services);
+            
+            switch (dbms)
+            {
+                case DBMS.MySQL:
+                {
+                    ConfigureForMySql(services, connectionString, dbms);
+                    break;
+                }
+                case DBMS.SQLite:
+                {
+                    ConfigureForSqLite(services, connectionString, dbms);
+                    break;
+                }
+                // todo: Add the case for MSSQL.
+                default:
+                {
+                    throw new ArgumentException("The database you have selected is not yet supported.");
```

Comment:
#Sadface

Reply:
@Krummelz 🤣 I will add this very soon. Just need to write some testing against it.